src/main/java/mate/academy/bookstore/model/Order.java

```diff
@@ -0,0 +1,57 @@
+package mate.academy.bookstore.model;
+
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.HashSet;
+import java.util.Set;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "orders")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "user_id", nullable = false)
+    private User user;
+    @Enumerated(value = EnumType.STRING)
+    @Column(nullable = false)
+    private OrderStatus orderStatus;
+    @Column(nullable = false)
+    private BigDecimal total;
+    @Column(nullable = false)
+    private LocalDateTime orderDate;
```

Comment:
you can initialise date here and remove initialising from service method
```suggestion
    private LocalDateTime orderDate = LocalDateTime.now();
```